miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -31,9 +31,12 @@ const.ERR_SETTING_MAP_ITEM_ON_NON_MAP_SLOT=0x00020048
 # Account procedure is not part of the account code
 const.ERR_PROC_NOT_PART_OF_ACCOUNT_CODE=0x0002004A
 
-# Provided index is out of bounds
+# Provided procedure index is out of bounds
 const.ERR_PROC_INDEX_OUT_OF_BOUNDS=0x0002004B
 
+# Provided storage slot index is out of bounds
+const.ERR_STORAGE_SLOT_INDEX_OUT_OF_BOUNDS=0x0002004E
```

Comment:
We should add an entry for this error in the `KERNEL_ERRORS` array of the `miden-tx/src/errors.rs` file.